website/config/custom.js

```diff
@@ -149,13 +149,21 @@ module.exports.custom = {
     'website/config/custom.js': 'mikermcneil',
 
     // 🦿 Handbook
-    'handbook/company/ceo.md': 'sampfluger88',
-    'handbook/company': 'mikermcneil',
-    'handbook/business-operations': 'mikermcneil',// TODO: Transfer to joanne once the philosophical stuff is moved into the new "Communication" page (similar to GitLab's)
-    'handbook/engineering': 'lukeheath',
-    'handbook/product': 'zhumo',
-    'handbook/customers': 'alexmitchelliii',
-    'handbook/marketing': 'jarodreyes',
+      'handbook/': 'mikermcneil',
+    // Everyone can contribute to the handbook, and page DRIs are expected to keep their pages up to date.  
```

Comment:
"See https://github.com/fleetdm/fleet/pull/13195/files"
Instead of dup comment.